scripts/cnv_wdl/somatic/README.md

```diff
@@ -49,4 +50,9 @@ The reference (and bins, if specified) used must be the same between PoN and cas
 
 In additional, there are several task-level parameters that may be set by advanced users as above.
 
+To invoke oncotator on the called tumor copy ratio segments:
+
+- ``CNVSomaticPairWorkflow.is_run_oncotator`` -- (optional) If true, run oncotator on the called copy ratio segments.  This will generate both a simple tsv and a gene list.
```

Comment:
oncotator -> Oncotator if done above.

Reply:
tsv -> TSV would be more consistent with the CNV Javadoc.

Reply:
Done